packages/engineer/src/cli-commands.ts

```diff
@@ -142,8 +142,6 @@ export function buildCommand(program: Command) {
         .option('--outDir <outDir>', 'output directory for the built application', 'dist-app')
         .option('--webpackConfig <webpackConfig>', 'path to webpack config to build the application with')
         .option('--publicConfigsRoute <publicConfigsRoute>', 'public route for configurations')
-        .option('--external [true|false]', 'build feature as external', parseBoolean, false)
-        .option('--eagerEntrypoints [true|false]', 'build feature as external', parseBoolean, false)
```

Comment:
this has nothing to do with external. it has to do with how the dynamic import is marked (for the entry point engine-scripts creates). see 'eager' @ https://webpack.js.org/api/module-methods/#magic-comments

could have been kept, but not crucial

Reply:
IMO It's best to cleanup at this case, and re-implement if needed